azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -89,6 +94,10 @@ public class ExecutableFlow extends ExecutableFlowBase {
   private String failedJobId = "unknown";
   private String modifiedBy = "unknown";
   private DispatchMethod dispatchMethod;
+  // how many times has flow level retry happened - retry defined by user's final status
+  private int customRetriedTimes = 0;
```

Comment:
should we consider this variable as `AtomicInt` or it could be an overkill here

Reply:
I think it's fine as no racing here